interfaces/builtin/login_session_observe.go

```diff
@@ -70,6 +70,12 @@ dbus (send)
     interface=org.freedesktop.DBus.Properties
     member=Get{,All},
 
+dbus (send)
+    bus=system
+    path=/org/freedesktop/login1{,/seat/*,/session/*,/user/*}
```

Comment:
LGTM, but since this includes the definition of the rule on line 62 above it, can it please be moved to replace the existing introspect rule there entirely? Thanks.